### PR TITLE
fix(misc): update readmes

### PR DESCRIPTION
**Pull request by juristr**
updates
- Nx GitHub repository readme
- Generated readmes for all the available Nx presets (via CNW). Here's a snapshot of the new readme for each preset that can be used: https://github.com/juristr/nx_readme_updates/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,23 +23,42 @@
...
 
-<p style="text-align: center;"><a href="https://nx.dev/#learning-materials" target="_blank" rel="noreferrer"><img src="./images/nx-courses-and-videos.svg" 
-width="100%" alt="Nx - Smart Monorepos · Fast CI"></a></p>
+...or run
+
+```
+npx nx init
+```
+
+to add Nx to your existing workspace to get faster task scheduling, caching and more. More [on the docs](https://nx.dev/getting-started/intro#try-nx-yourself).
```
Comment (jaysoo) [inline README.md:38]: "more in the docs"

File: README.md [hunk 2]
```diff
@@ -23,23 +23,42 @@
...
 
-- [Nx.Dev Community Page: Community Discord Channel, Newsletter, etc.](https://nx.dev/community)
-- [The Nx Show Playlist on YouTube](https://www.youtube.com/playlist?list=PLakNactNC1dE8KLQ5zd3fQwu_yQHjTmR5). It's a
-  regular YouTube stream where we talk all things Nx. Join the stream, ask questions, etc.
-- [Follow Nx on Twitter](https://twitter.com/NxDevTools)
+```
+npx nx connect
+```
+
+Learn more on the [Nx CI docs &raquo;](https://nx.dev/ci/intro?utm_source=nxrepo&utm_medium=readme&utm_campaign=nxrepo)
```
Comment (jaysoo) [inline README.md:50]: "Learn more in the Nx CI docs"

File: README.md [hunk 3]
```diff
@@ -23,23 +23,42 @@
...
+
+```
+npx nx init
+```
+
+to add Nx to your existing workspace to get faster task scheduling, caching and more. More [on the docs](https://nx.dev/getting-started/intro#try-nx-yourself).
+
+## Learn about CI with Nx Cloud
+
+[Nx Cloud](https://nx.dev/nx-cloud) directly connects to your existing CI setup, helping you scale your monorepos on CI by leveraging [remote caching](https://nx.dev/ci/features/remote-cache?utm_source=nxrepo&utm_medium=readme&utm_campaign=nxrepo), [task distribution across multiple machines](https://nx.dev/ci/features/distribute-task-execution?utm_source=nxrepo&utm_medium=readme&utm_campaign=nxrepo), [automated e2e test splitting](https://nx.dev/ci/features/split-e2e-tasks?utm_source=nxrepo&utm_medium=readme&utm_campaign=nxrepo) and [automated task flakiness detection](https://nx.dev/ci/features/flaky-tasks?utm_source=nxrepo&utm_medium=readme&utm_campaign=nxrepo)
```
Comment (jaysoo) [inline README.md:42]: I'd go with "Nx Cloud connects directly to your existing CI setup" or just "Nx Cloud connects to your existing CI setup".

File: packages/workspace/src/generators/new/files-readme/README.md.template [hunk 4]
```diff
@@ -2,88 +2,148 @@
 
 <a alt="Nx logo" href="https://nx.dev" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-logo.png" width="45"></a>
 
-✨ **This workspace has been generated by [Nx, Smart Monorepos · Fast CI.](https://nx.dev)** ✨
+✨ Your new, shiny [Nx workspace](https://nx.dev) is ready ✨. 
+<% if (learnMoreLink) { %>
+[Learn more about this workspace setup and its capabilities](<%= learnMoreLink %>) or run `npx nx graph` to visually explore what got created. Now, let's get you up to speed!
```
Comment (jaysoo) [inline packages/workspace/src/generators/new/files-readme/README.md.template:7]: "explore what was created"

File: packages/workspace/src/generators/new/files-readme/README.md.template [hunk 5]
```diff
@@ -2,88 +2,148 @@
...
+To version and release the library use
 
-# View project details
-npx nx show project demo --web
+```
+npx nx release
 ```
 
-Run `npx nx list` to get a list of available plugins and whether they have generators. Then run `npx nx list <plugin-name>` to see what generators are available.
+Feel free to add `--dry-run` to see what would happen without actually releasing the library.
```
Comment (jaysoo) [inline packages/workspace/src/generators/new/files-readme/README.md.template:64]: "Pass `--dry-run` to see`. Less passive phrasing.

File: README.md (hunk 2 above)
Comment (juristr) [reply README.md:50]: idk what I had with this "on the docs" thingy today morning 😬 . but thanks for catching

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

File: packages/angular/src/generators/library/files/base/README.md__tpl__ [hunk 6]
```diff
@@ -7,3 +7,20 @@ This library was generated with [Nx](https://nx.dev).
...
+
+```
+nx show project <%= libName %>
+```
+
+Alternatively, use [Nx Console](https://nx.dev/getting-started/editor-setup?utm_source=nx_project&utm_medium=readme&utm_campaign=nx_projects) to view the tasks right in your editor.
+
+## Publish your library to NPM
+
+If you're planning to version and publish your library to NPM, check out the [nx release](https://nx.dev/features/manage-releases?utm_source=nx_project&utm_medium=readme&utm_campaign=nx_projects) command.
```
[the remark this answers is not in the document]
Comment (juristr) [reply packages/angular/src/generators/library/files/base/README.md__tpl__:24]: let's remove library READMEs for now and I'll follow up with a separate PR. They are less relevant as they're seen less often.

File: packages/workspace/src/generators/new/files-readme/README.md.template [hunk 7]
```diff
@@ -2,88 +2,141 @@
...
+npx nx connect
+```
+
+This gives you:
 
-Nx comes with local caching already built-in (check your `nx.json`). On CI you might want to go a step further.
+- [Remote caching](https://nx.dev/ci/features/remote-cache?utm_source=nx_project&utm_medium=readme&utm_campaign=nx_projects)
+- [Task distribution across multiple machines](https://nx.dev/ci/features/distribute-task-execution?utm_source=nx_project&utm_medium=readme&utm_campaign=nx_projects)
+- [Automated e2e test splitting](https://nx.dev/ci/features/split-e2e-tasks?utm_source=nx_project&utm_medium=readme&utm_campaign=nx_projects)
+- [Task flakiness detection and rerunning](https://nx.dev/ci/features/flaky-tasks?utm_source=nx_project&utm_medium=readme&utm_campaign=nx_projects)
```
[the remark this answers is not in the document]
Comment (juristr) [reply packages/workspace/src/generators/new/files-readme/README.md.template:121]: @FrozenPandaz I pushed all changes. that we agreed on. For the setup CI part Victor mentioned you're going to update them anyway so feel free to take over this PR or merge it in and update with the remaining changes. LMK if I can help

**Review by FrozenPandaz — APPROVED** [review]

**Review by jaysoo — APPROVED** [review]

**Pull request merged**